app/views/default.scala.html

```diff
@@ -35,6 +36,8 @@
           nbextensions : '/nbextensions',
           kernelspecs : '/kernelspecs',
           underscore : '@routes.Assets.at("javascripts/third/underscore-min")',
+          taucharts_root: '@routes.Assets.at("javascripts/taucharts")',
+          tauCharts : '@routes.Assets.at("javascripts/taucharts/tauCharts")',
```

Comment:
it might be we need shim to define dependencies of tauCharts (d3, underscore).

but works fine without too...